Source/Examples/Example01_windows_blit_tree/Main.cpp

```diff
@@ -0,0 +1,205 @@
+#include "Application.h"
+#include "FrameBuffer.h"
+#include "Timer.h"
+#include <iostream> 
+#include <math.h>
+#include <thread>
+
+#define PI 3.14159265
+
+void drawTree();
+void drawBranch(int treeLevel,int rootX, int rootY, int branchWidth, int branchHeight, int degree);
+
+void renderMain();
+
+
+constexpr int imageWidth = 800;
+constexpr int imageHeight = 800;
+constexpr int pixelNumber = imageHeight * imageWidth;
+FrameBuffer<PixelFormat::BGRA> frameBuffer(nullptr, imageWidth, imageHeight);
+
+static XYZ pixelColor(255, 0, 0);
+static uint8_t pixelAlpha = 255;
+static int pixelPosX = 400;
+static int pixelPosY = 400;
+
+static XYZ truckColor(165, 42, 0); //�����֦����ɫ
+static XYZ leafColor(0, 255, 0); //��Ҷ����ɫ
+constexpr int truckWidth = 28; //��ɵĿ��
+constexpr int truckHeight = 200; //��ɵĸ߶�
+constexpr int truckPosX = imageWidth/2;  //���������Ĵ������X(��1����辿��洦)
+constexpr int truckPosY = imageHeight-truckHeight;  //���������Ĵ������Y(��1����辿��洦)
+constexpr float branchWidthRatio = 0.55f;  //n+1�����֦��n�����֦�ֶȵı�ֵ����֦Խ����Խϸ
+constexpr float branchHeightRatio = 0.75f; //n+1�����֦��n�����֦���ȵı�ֵ����֦Խ����Խ��
+constexpr int branchDegree = 19; //n+1�����֦��n�����֦�ļн�
+constexpr int HighestLevel = 7; //������ܳɳ����ٲ�
+
+bool isAnimationPlay = true; //�Ƿ񲥷���Ķ�����true�����Ŷ�����false��ֱ����ɽ��
+constexpr float aTimeInterval = 0.5f; //���Ŷ���ʱ��ÿ���һ����֦��ʱ����
+Timer aTimer;
```

Comment:
I understand why you added it. But it is not a good idea to have timer codes in a draw algorithm. We should split them. You can reference my codes to have a render thread. And you can tick this render thread slowly.